packages/server/src/koaJsonRpc/index.ts

```diff
@@ -104,21 +115,26 @@ export default class KoaJsonRpc {
       ) {
         ctx.body = jsonResp(body.id || null, new InvalidRequest(), undefined);
         ctx.status = 400;
+        ms = Date.now() - this.startTimestamp;
+        this.logger.error(`${messagePrefix} ${ctx.status} (INVALID REQUEST) ${ms} ms`);
```

Comment:
nit: `INVALID REQUEST`, 1METHOD NOT FOUND` and the rest should be const class variables.

Reply:
Updated.